DevelopedCalculator.js

```diff
@@ -13,14 +13,20 @@ class DevelopedCalculator extends Calculator {
             this.historyDiv.appendChild(document.createTextNode("History:" + this.currentResult));
             // prepend the historyDiv in the host element 
             this.hostElem.prepend(this.historyDiv);
+            //this.historyDiv.gettingHistoryArray();
         }
         // new method has a parameter
     onButtonNumberClick(event) {
         // calling parent class's onButtonNumberClick method
         super.onButtonNumberClick(event);
         // pushing the history array with current result
         this.history.push(this.currentResult);
+        //this.historyDiv.gettingHistoryArray();
```

Comment:
can you explain what `historyDiv` is?

Reply:
It is the div element where we can seek the history in it

Reply:
does `div` elements have `gettingHistoryArray`  method?

Reply:
No they do not 
I have pushed some updates before reading this comment